shacl12-ui/index.html

```diff
@@ -387,6 +387,73 @@ <h3>Terminology</h3>
           </dfn>
           .
         </p>
+        <div class="def">
+          <div class="term-def-header">Language Resolution</div>
+          <div>
+            <dfn data-lt="language resolution">Language resolution</dfn> is the process of determining and ordering the
```

Comment:
```suggestion
            <dfn data-lt="language resolution">Language Resolution</dfn> is the process of determining and ordering the
```

Added upper-case so it's consistent with the other definitions.